File: rps.js
// Rock, Paper, Scissors Game
const getUserChoice = userInput =>  
{if(userInput === 'rock'|userInput==='paper'|userInput==='scissors'){
  return userInput
  console.log(userInput)
}else if(userInput != userInput.toLowerCase()){
  return userInput.toLowerCase()
}else{
  console.log('Invalid Entry.')
}};


function getComputerChoice(){
    Math.floor(Math.random() * 2);
    if(getComputerChoice === 0){
        return 'rock'
    }else if(getComputerChoice === 1){
        return 'paper'
    }else{
        return 'scissors'
    }}

function determineWinner(userChoice,computerChoice){
    if(userChoice===computerChoice){
        return "It's a tie!"
    }

    if(userChoice === 'rock'){
      if(computerChoice === 'paper'){
        return 'Computer Wins :('
      }else{
        return 'User Wins!'
      }
    }

    if(userChoice==='paper'){
      if(computerChoice==='scissors'){
        return 'Computer Wins :('
      }else{
        return 'User Wins!'
      }
    }

    if(userChoice==='scissors'){
      if(computerChoice==='rock'){
        return 'Computer Wins :('
      }else{
        return 'User Wins!'
      }
    }
    // userChoice === 'rock' && (computerChoice === 'paper') ? 
    // console.log ('Computer wins :('): 
    // console.log('User Wins!');

    // userChoice === 'paper' && (computerChoice === 'scissors') ? 
    // console.log ('Computer wins :('): 
    // console.log('User Wins!');
    
    // userChoice === 'scissors' && (computerChoice === 'rock') ? 
    // console.log ('Computer wins :('): 
    // console.log('User Wins!');
}

function playGame(){
  let userChoice = getUserChoice('rock')
  console.log(userChoice)
  let computerChoice = getComputerChoice()
  console.log(computerChoice)
  console.log(determineWinner(userChoice,computerChoice))
}

playGame()

File: rps.py
#Rock, paper, scissors game
import random

get_user_choice = input('Pick rock, paper, or scissors: ')
def user_choice():
    choice = get_user_choice
    if (choice == choice.upper()) or (choice == choice.title()):
        print(f"User's choice: {choice.lower()}")
    elif choice != get_user_choice:
        print("Invalid entry")
    else:
        print(f"User's choice: {choice}")
    

    

def comp_choice():
    choice = ['rock','paper','scissors']
    return random.choice(choice)

print(f"Computer's choice: {comp_choice()}")

#Fix winner control flow
def winner():
    user_choice()
    comp_choice()
    if user_choice == comp_choice:
        return "It's a tie!"
    elif (user_choice == 'rock') & (comp_choice == 'paper'):
        return "Computer Wins"
    elif(user_choice=='paper') & (comp_choice=='scissors'):
        return "Computer Wins"
    elif(user_choice=='scissors') & (comp_choice=='rock'):
        return "Computer Wins"
    else:
        return "User Wins"

print(winner())

